### DISCUSSION TITLE
How to get term list in a term page?

### DISCUSSION THREAD

**Discussion by tomasg-lt** [statamic/cms#5868] [Q&A] [answered]
Hi,
I have this kind of structure:

Taxonomy USED HARDWARE (blueprint used_hardware) with terms:
- CPU
- GPU
- Memory
- ... etc.

Taxonomy NEW HARDWARE (blueprint new_hardware) with terms:
- CASES
- COOOLERS
- CPU
- ... etc...

I am able to display all the USED HARDWARE term tree while I'm in index.antlers.html of the taxonomy (for example I'm in USED HARDWARE page) just by using {{ terms }} , but I'm not sure how to display all the USED HARDWARE term tree while I'm in actual Term page show.antlers.html (for example USED HARDWARE >> CPU).

How can I display all the terms from the same Taxonomy while I'm in a term page?

Thanks!

**Comment by jasonvarga** [comment, marked as answer]
You only get the automatic `terms` variable when you're on that taxonomy's page.

For all the other times, you can use the taxonomy tag. https://statamic.dev/tags/taxonomy

```antlers
{{ taxonomy:used_hardware }}
  <a href="{{ url }}">{{ title }}</a>
{{ /taxonomy:used_hardware }}
```

**Reply by tomasg-lt** [reply]
I have blueprint name in a term - {{ blueprint }}
Is there a way to call taxonomy dynamically by blueprint name? For example:

`{{ taxonomy:BLUEPRINT NAME HERE }}
  <a href="{{ url }}">{{ title }}</a>
{{ /taxonomy:BLUEPRINT NAME HERE }}`

**Reply by tomasg-lt** [reply]
I mean to get all terms which have the same parent Taxonomy without creating templates for all the term template files manually. 
So even my client creates a new Term in USED HARDWARE taxonomy it will display it's Taxonomy term list automatically in term's show.antlers template.

{{ taxonomy from="{{ blueprint }}" }} definitely will not work but I need something similar.

**Reply by tomasg-lt** [reply]
Sorry, that did solve the issue, just needed to use {{ taxonomy from=blueprint }} .

Thanks!

**Reply by jasonvarga** [reply]
`{{ taxonomy :from="blueprint" }}`
